### PR TITLE
removed scaler from deployment

### PR DESCRIPTION
**Pull request by yzainee-zz**
removed scaler from deployment

### CODE REVIEW

File: openshift/deploy.sh [hunk 1]
```diff
@@ -79,8 +79,8 @@ oc_process_apply "${templates_dir}/worker.yaml" "-p WORKER_ADMINISTRATION_REGION
 oc_process_apply "${templates_dir}/worker.yaml" "-p WORKER_ADMINISTRATION_REGION=api -p WORKER_INCLUDE_QUEUES=GraphImporterTask -p WORKER_NAME_SUFFIX=-graph-import"
 oc_process_apply "${templates_dir}/f8a-server-backbone.yaml"
 oc_process_apply "${templates_dir}/server.yaml"
-oc_process_apply "${templates_dir}/scaler.yaml" "-p DC_NAME=bayesian-worker-ingestion -p SQS_QUEUE_NAME=ingestion_bayesianFlow_v0 -p MAX_REPLICAS=8 -p DEFAULT_REPLICAS=2"
-oc_process_apply "${templates_dir}/scaler.yaml" "-p DC_NAME=bayesian-worker-api -p SQS_QUEUE_NAME=api_bayesianFlow_v0 -p MAX_REPLICAS=4 -p DEFAULT_REPLICAS=2"
+#oc_process_apply "${templates_dir}/scaler.yaml" "-p DC_NAME=bayesian-worker-ingestion -p SQS_QUEUE_NAME=ingestion_bayesianFlow_v0 -p MAX_REPLICAS=8 -p DEFAULT_REPLICAS=2"
+#oc_process_apply "${templates_dir}/scaler.yaml" "-p DC_NAME=bayesian-worker-api -p SQS_QUEUE_NAME=api_bayesianFlow_v0 -p MAX_REPLICAS=4 -p DEFAULT_REPLICAS=2"
```
Comment (msrb) [inline openshift/deploy.sh]: I think we can remove these lines completely. It is unlikely that developers will need this service, and simply uncommenting would not work as we are removing the service from the list above.

**Review by msrb — CHANGES_REQUESTED** [review]
Thanks @yzainee :wink: Just a small nitpick, LGTM otherwise.

**Comment by yzainee-zz** [comment]
@msrb done

**Review by msrb — APPROVED** [review]
LGTM, thanks :wink:

**Pull request merged**